airbyte-cdk/python/airbyte_cdk/sources/declarative/interpolation/jinja.py

```diff
@@ -3,19 +3,38 @@
 #
 
 import ast
+from typing import Optional
 
 from airbyte_cdk.sources.declarative.interpolation.interpolation import Interpolation
 from airbyte_cdk.sources.declarative.interpolation.macros import macros
+from airbyte_cdk.sources.declarative.types import Config
 from jinja2 import Environment
 from jinja2.exceptions import UndefinedError
 
 
 class JinjaInterpolation(Interpolation):
+    """
+    Interpolation strategy using the Jinja2 template engine.
+
+    If the input string is a raw string, the interpolated string will be the same.
+    `eval("hello world") -> "hello world"`
+
+    The engine will evaluate the content passed within {{}}, interpolating the keys from the config and kwargs passed as argument.
```

Comment:
Remember that users of this are not necessarily python coders, so we should not use very pythonic words like `kwargs`
```suggestion
    The engine will evaluate the content passed within {{}}, interpolating the keys from the config and context-specific arguments.
```